src/ExampleClass.h

```diff
@@ -26,7 +26,7 @@ class ExampleClass : public EventSystem<ExampleClass> {
 
         //Now let's say we want to send an event...
         EventSystem::queueEvent(
-            Event("someKindofEvent",
+            Event(EventType::load,
```

Comment:
For the example we should probably use some dummy event type to avoid clashes with real things. My suggestion, EXAMPLE_DUMMY_EVENT